feature-libs/user/schematics/add-user/index.ts

```diff
@@ -116,5 +112,11 @@ function addProfileFeature(options: SpartacusUserOptions): Rule {
       scssFileName: SCSS_FILE_NAME,
       importStyle: SPARTACUS_USER,
     },
+    dependencyManagement: {
+      featureName: CLI_USER_FEATURE,
```

Comment:
Should we use here the user profile feature? As the dependencies are for specific features, not libraries

Reply:
I agree. Done.